packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/invoke-contract-v2.test.ts

```diff
@@ -125,7 +125,7 @@ test("Quorum Ledger Connector Plugin", async (t: Test) => {
       "contractAddress typeof string OK",
```

Comment:
@AzaharaC I'd also rename this file to drop the v2 because once we hit the v2 major release of Cactus (in the future) then this will collide name-wise with the tests that we'll add then.

Reply:
done